### ISSUE TITLE
Positive Pixel Count error

### ISSUE THREAD

**Issue by yliu7366** [DigitalSlideArchive/HistomicsTK#455]
Hi,

I followed the example (https://digitalslidearchive.github.io/HistomicsTK/examples/positive-pixel-count.html) and used the example image under the section of "Analyzing a small image with count_image" but got some error messages:

INFO:root:Using
python for large_image caching Traceback (most recent call last): File "PositivePixelCount/PositivePixelCount.py", line 49, in <module> main(CLIArgumentParser().parse_args()) File "PositivePixelCount/PositivePixelCount.py", line 39, in main label_image = color_map[label_image]
IndexError: index 4 is out of bounds for axis 0 with size 4

This same error happens if I try my own images.

I was using a fresh installation of HistomicsTK and my test was in the HistomicsTK web interface instead of a notebook.

**Comment by cdeepakroy** [comment]
@yliu7366 Thanks for reporting the issue. I will take a look at this early next week.

**Comment by cdeepakroy** [comment]
@yliu7366 I just tested both the jupyter notebook example for positive pixel count and running through the UI. In both cases i was able to get the expected result

![image](https://user-images.githubusercontent.com/1923534/35987493-cd598ef8-0cc9-11e8-9a8b-fb5f273deb89.png)

Could you tell me the image and the parameters that you used so i can replicate the error?

**Comment by yliu7366** [comment]
@cdeepakroy thank you very much for looking into this issue. I am using the same image (screen copied from the jupyter notebook at resolution of 1688x932) and the same parameters in the notebook example. One thing I did is to convert the image to be a large_image format as HistomicsTK complains about 'not a large_image format' when I tried to open the image as is. Other than that I can not think about something else related to image index (size) error.
